src/index.html

```diff
@@ -10,23 +10,28 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Two Search Bars</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search-conteiner">
+      <input
+        type="text"
+        class="search-bar big"
+        placeholder="Search..."
+        data-qa="big"
+        data-qa-keypress=""
```

Comment:
The 'data-qa-keypress' attribute is empty. Ensure this is intentional or provide a value if needed.